dotty-scala-library-from-tasty-tests/test/LibraryTest.scala

```diff
@@ -0,0 +1,292 @@
+    "scala.collection.BitSetOps",
+    "scala.collection.generic.DefaultSerializationProxy",
+    "scala.collection.immutable.LazyList",
+    "scala.collection.immutable.List",
+    "scala.collection.immutable.Node",
+    "scala.collection.immutable.NumericRange",
+    "scala.collection.immutable.StrictOptimizedSeqOps",
+    "scala.collection.immutable.TreeSeqMap",
+    "scala.collection.immutable.Vector",
+    "scala.collection.immutable.Vector0",
+    "scala.collection.immutable.VectorBuilder",
+    "scala.collection.immutable.VectorSliceBuilder",
+    "scala.collection.immutable.WrappedString",
+    "scala.collection.LazyZip2",
+    "scala.collection.LazyZip3",
+    "scala.collection.LazyZip4",
+    "scala.collection.MapOps",
+    "scala.collection.mutable.ArrayBuffer",
+    "scala.collection.mutable.ArrayBufferView",
+    "scala.collection.mutable.ArrayDeque",
+    "scala.collection.mutable.ArrayDequeOps",
+    "scala.collection.mutable.CollisionProofHashMap",
+    "scala.collection.mutable.HashMap",
+    "scala.collection.mutable.HashSet",
+    "scala.collection.mutable.ListBuffer",
+    "scala.collection.mutable.UnrolledBuffer",
+    "scala.collection.SeqView",
+    "scala.collection.StringView",
+    "scala.concurrent.duration.Duration",
+    "scala.concurrent.Future",
+    "scala.Enumeration",
+    "scala.io.Source",
+    "scala.jdk.Accumulator",
+    "scala.jdk.DoubleAccumulator",
+    "scala.jdk.IntAccumulator",
+    "scala.jdk.javaapi.DurationConverters",
+    "scala.jdk.LongAccumulator",
+    "scala.Product",
+    "scala.Product1",
+    "scala.Product10",
+    "scala.Product11",
+    "scala.Product12",
+    "scala.Product13",
+    "scala.Product14",
+    "scala.Product15",
+    "scala.Product16",
+    "scala.Product17",
+    "scala.Product18",
+    "scala.Product19",
+    "scala.Product2",
+    "scala.Product20",
+    "scala.Product21",
+    "scala.Product22",
+    "scala.Product3",
+    "scala.Product4",
+    "scala.Product5",
+    "scala.Product6",
+    "scala.Product7",
+    "scala.Product8",
+    "scala.Product9",
+    "scala.reflect.ClassTag",
+    "scala.runtime.ArrayCharSequence",
+    "scala.runtime.LazyBoolean",
+    "scala.runtime.LazyByte",
+    "scala.runtime.LazyChar",
+    "scala.runtime.LazyDouble",
+    "scala.runtime.LazyFloat",
+    "scala.runtime.LazyInt",
+    "scala.runtime.LazyLong",
+    "scala.runtime.LazyRef",
+    "scala.runtime.LazyShort",
+    "scala.runtime.LazyUnit",
+    "scala.runtime.Tuple2Zipped",
+    "scala.runtime.Tuple3Zipped",
+    "scala.StringContext",
+    "scala.sys.process.ProcessImpl",
+
+    // undefined: _root_.scala.StringContext.apply(["Scala "," "," -- ","" : String]:String*).f # -1: TermRef(TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class <root>)),module scala),class StringContext),f) at readTasty
+    "scala.collection.immutable.Range",
+    "scala.collection.Iterator",
+    "scala.util.PropertiesTrait",
+  )
+
+  def compileBlacklist = List[String](
```

Comment:
We already compile the stdlib as part of the community build, so how come we're seeing compile failures here?

Reply:
I added some docs

Reply:
Those recompilations from the tasty of the stdlib